week05/my-todolist/src/App.js

```diff
@@ -1,32 +1,42 @@
-import { useEffect, useState } from "react";
-import Header from "./Header";
-import TodoList from "./TodoList";
-import TodoAdd from "./TodoAdd";
-import StateTest from "./StateTest";
+import { Routes, Route, NavLink } from "react-router-dom";
+//import StateTest from "./StateTest";
 import "./App.css";
-
-const TODO_LIST_KEY = "todoList";
+import MyTodolist from "./components/pages/MyTodolist";
+import MyPlaylist from "./components/pages/MyPlaylist";
 
 function App() {
```

Comment:
4. state를 만들고 기본값을 랜딩 페이지에 맞게 설정합니다.

```suggestion
function App() {
const [bgImg, setBgImg] = useState(bgTodolist);
```